src/material-examples/focus-monitor-focus-via/focus-monitor-focus-via-example.ts

```diff
@@ -16,7 +16,7 @@ import {
   styleUrls: ['focus-monitor-focus-via-example.css']
 })
 export class FocusMonitorFocusViaExample implements OnDestroy, OnInit {
-  @ViewChild('monitored') monitoredEl: ElementRef<HTMLElement>;
+  @ViewChild('monitored', {static: true}) monitoredEl: ElementRef<HTMLElement>;
```

Comment:
Should we move this into `ngOnInit` instead? We probably shouldn't be encouraging people to use static queries.

Reply:
Done